app/test-utils/componentTester.js

```diff
@@ -0,0 +1,16 @@
+import nunjucks from 'nunjucks';
+import { createTestHarness } from 'buying-catalogue-library';
+import { App } from '../app';
+import config from '../config';
+
+export const componentTester = (setup, callback) => (done) => {
+  const app = new App().createApp();
+
+  callback(createTestHarness({
+    app,
+    templateEngine: nunjucks,
```

Comment:
will this not be the same for everything? can we set it in the `createTestHarness` function?

Reply:
Yeah this was the tricky bit. If we move this into BCL then we need to set up nunjucks in BCL too which will be a pain.